libraries/AC_PID/AC_HELI_PID.cpp

```diff
@@ -48,16 +48,16 @@ const AP_Param::GroupInfo AC_HELI_PID::var_info[] = {
 };
 
 /// Constructor for PID
-AC_HELI_PID::AC_HELI_PID(float initial_p, float initial_i, float initial_d, float initial_imax, float initial_filt_hz, float dt, float initial_ff) :
-    AC_PID(initial_p, initial_i, initial_d, initial_imax, initial_filt_hz, dt, initial_ff)
+AC_HELI_PID::AC_HELI_PID(float initial_p, float initial_i, float initial_d, float initial_ff, float initial_imax, float initial_filt_T_hz, float initial_filt_E_hz, float initial_filt_D_hz, float dt) :
+    AC_PID(initial_p, initial_i, initial_d, initial_ff, initial_imax, initial_filt_T_hz, initial_filt_E_hz, initial_filt_D_hz, dt)
 {
     _last_requested_rate = 0;
 }
 
 // This is an integrator which tends to decay to zero naturally
 // if the error is zero.
 
-float AC_HELI_PID::get_leaky_i(float leak_rate)
+void AC_HELI_PID::get_leaky_i(float leak_rate)
```

Comment:
the function name is a bit weird.  normally a function starting with "get_" will return something but here it doesn't.

Reply:
Changed to update_leaky_i